Nitrox.Launcher/Models/Utils/GameInspect.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using HanumanInstitute.MvvmDialogs;
+using Nitrox.Launcher.ViewModels;
+using NitroxModel.Helper;
+using NitroxModel.Logger;
+using NitroxModel.Platforms.OS.Shared;
+
+namespace Nitrox.Launcher.Models.Utils;
+
+internal static class GameInspect
+{
+    /// <summary>
+    ///     Check to ensure the Subnautica is not in legacy.
+    /// </summary>
+    public static async Task<bool> IsOutdatedGameAndNotify(string gameInstallDir, IDialogService dialogService = null)
+    {
+        try
+        {
+            ArgumentException.ThrowIfNullOrWhiteSpace(gameInstallDir);
+            
+            string gameVersionFile = Path.Combine(gameInstallDir, GameInfo.Subnautica.DataFolder, "StreamingAssets", "SNUnmanagedData", "plastic_status.ignore");
+            if (int.TryParse(await File.ReadAllTextAsync(gameVersionFile), out int gameVersion) && gameVersion <= 68598)
```

Comment:
Maybe the version is something we might wanna have in GameInfo or whatever ?

Reply:
Think I'll leave this as is for. This code isn't proper for GameInfo support as is and want to see what HeN is doing.